w06h02/test/pgdp/ds/MultiStackTest.java

```diff
@@ -202,7 +202,7 @@ void topShouldReturnCorrectValues() {
 
         assertArrayEquals(array(value), getMem(getHead()));
         assertArrayEquals(array(value2, value3), getMem(getNext(getHead())));
-        assertArrayEquals(array(value4, 0, 0, 0), getMem(getNext(getNext(getHead()))));
+        assertEquals(value4, getMem(getNext(getNext(getHead())))[0]);
```

Comment:
Same here and by the way: could you maybe extract this to a new variable for readability purposes?

Reply:
👍